dbt/contracts/graph/parsed.py

```diff
@@ -160,6 +195,39 @@
 )
 
 
+# The parsed node update is only the 'patch', not the test. The test became a
+# regular parsed node. Note that description and columns must be present, but
+# may be empty.
+PARSED_NODE_PATCH_CONTRACT = {
+    'type': 'object',
+    'additionalProperties': False,
+    'description': 'A collection of values that can be set on a node',
+    'properties': {
+        'name': {
+            'type': 'string',
+            'description': 'The name of the node this modifies',
+        },
+        'description': {
+            'type': 'string',
+            'description': 'The description of the node to add',
+        },
+        'path': {
+            'type': 'string',
+            'description': 'The original file path the patch came from',
+        },
```

Comment:
for consistency, can you change this field to look like the node version:

```
'original_file_path': {
            'type': 'string',
            'description': (
                'Relative path to the originating file from the project root.'
                ),
        },
```

i was reading code related to his below and got very confused by what "path" was

Reply:
Sure. I'm trying to highlight the distinction here, the patch path becomes `patch_path` on the node when it's applied.